modules/renter/repair.go

```diff
@@ -290,14 +288,7 @@ func (r *Renter) managedRepairIteration(rs *repairState) {
 			}
 		}
 
-		// Don't update gaps for downloaded chunks twice
 		numGaps := chunkStatus.numGaps(rs)
-		if !downloading {
-			// Update the number of gaps for this chunk.
-			rs.gapCounts[chunkStatus.recordedGaps]--
-			rs.gapCounts[numGaps]++
-			chunkStatus.recordedGaps = numGaps
-		}
```

Comment:
pretty sure that this code is critical code, I don't think you want it removed.

Reply:
good rule of thumb for unstable opaque code is to assume that everything is important in delicate an unknown ways. I don't really remember why I put this here,  but I do remember that the gap count stuff was both tricky and also critical